address-controller/src/main/resources/swagger.json

```diff
@@ -0,0 +1,640 @@
+                    {
+                        "in": "body",
+                        "name": "body",
+                        "description": "AddressSpace object",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/AddressSpace"
+                        }
+                    }
+                ],
+                "responses": {
+                    "405": {
+                        "description": "Invalid input"
+                    }
+                }
+            }
+        },
+        "/addressspaces/{name}": {
+            "get": {
+                "tags": [
+                    "addressspaces"
+                ],
+                "summary": "Get AddressSpace by name",
+                "description": "Returns a single AddressSpace",
+                "operationId": "getAddressSpace",
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "name",
+                        "in": "path",
+                        "description": "Name of AddressSpace to return",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "successful operation",
+                        "schema": {
+                            "$ref": "#/definitions/AddressSpace"
+                        }
+                    },
+                    "404": {
+                        "description": "AddressSpace not found"
+                    }
+                }
+            },
+            "delete": {
+                "tags": [
+                    "addressspaces"
+                ],
+                "summary": "Delete an AddressSpace",
+                "description": "",
+                "operationId": "deleteAddressSpace",
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "name",
+                        "in": "path",
+                        "description": "Name of AddressSpace to delete",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Deletion successful; List of remaining AddressSpaces is returned.",
+                        "schema": {
+                            "$ref": "#/definitions/AddressSpaceList"
+                        }
+                    },
+                    "404": {
+                        "description": "AddressSpace not found"
+                    }
+                }
+            }
+        },
+        "/addresses/{addressSpace}": {
+            "get": {
```

Comment:
Wouldn't this make more sense as: /addressspaces/{name}/addresses

Reply:
Yes, we'll probably change that in the future.